doc/nrf/ug_thingy53.rst

```diff
@@ -214,13 +134,15 @@ To build and program the source code from the command line, complete the followi
    .. parsed-literal::
       :class: highlight
 
-      west build -b thingy53_nrf5340_cpuapp -d *destination_directory_name*
+      west build -b *build_target* -d *destination_directory_name*
+
+   The build target should be ``thingy53_nrf5340_cpuapp`` or ``thingy53_nrf5340_cpuapp_ns`` when building samples for the application core and ``thingy53_nrf5340_cpunet`` when building for the network core.
```

Comment:
Should we state that in most cases we use ``thingy53_nrf5340_cpuapp`` or ``thingy53_nrf5340_cpuapp_ns`` and cpunet firmware is built and flashed as a child image in the NCS ?

Reply:
That is already mentioned once on Lin 63, not sure whether or not it needs to be repeated here.

Reply:
In general we do not use `thingy53_nrf5340_cpunet` as a target board in Thingy:53 compatible samples (the netcore firmware is built by multiimage build system). Still the `thingy53_nrf5340_cpunet` board could be used e.g. with external programmer. Let's leave it assuming that user is aware what he's doing :)